### ISSUE TITLE
Replace dummy audience/feature values with real values from Nimbus Shared

### ISSUE THREAD

**Issue by jaredlockhart** [mozilla/experimenter#2931]
When the audience and feature values land in Nimbus Shared and the Python package is updated, we can replace our dummy values with the values it emits, both on the server and frontend.

Once we do this if we consume that library from both back/front end we will need to setup some kind of CI check to make sure both those packages are in sync (same verison).

**Comment by jaredlockhart** [comment]
Blocked on https://github.com/mozilla/nimbus-shared/issues/60

**Comment by jaredlockhart** [comment]
Blocked on https://github.com/mozilla/nimbus-shared/milestone/3

**Comment by jaredlockhart** [comment]
Audience:

- Frontend
  - Import shared repo as js dependency
  - Use shared repo audience constnats as UI constants
  - Pass audience slug to teh V3 API on save 
- Backend
  - Import shared repo as py dependency
  - Use audience for model/serializer choices
  - Use audience targeting when serializing recipe


Feature:
- Frontend
  - Import shared repo as js depency
  - Use shared repo feature constants as UI constants
  - Pass feature slug to the V3 API on save
- Backend
  - Import shared repo as py dependnecy
  - Use feature for model/serializer choices
  - Expose feature slugs in V4 API ( already done)
